### DISCUSSION TITLE
Archiving projects

### DISCUSSION THREAD

**Discussion by practicalwellbeing** [stacks-task-manager/stacks2#44] [Q&A] [answered]
Is there a way to archive projects after they are completed to keep the workspace clean.

I can't see anything obvious in the app or docs

If it is possible to move projects / folders between workspaces I guess I can create an Archive workspace and shift everything to that.

BTW I really like the Stacks

**Comment by synapse** [comment]
Hey @practicalwellbeing, there's no project archiving just yet, it's on the `to do` list.
There isn't any move or copy across workspaces either, since they might also be remote or in the cloud (via a syncing service) which would make kinda difficult to move or copy. But there will be an export/import feature which will be useful for these exact use cases.

**Reply by practicalwellbeing** [reply]
OK thanks. I look forward to it. There's plenty of time I haven't finished any projects yet!

**Comment by synapse** [comment, marked as answer]
This feature is now available since [version 2.3.0](https://getstacksapp.com/updates/#version-2.3.0)
